lib/bakeware/assembler.ex

```diff
@@ -184,4 +187,23 @@ defmodule Bakeware.Assembler do
 
     %{assembler | start_command: command}
   end
+
+  defp check_invalid_option!(release, option) do
+    if value = release.options[option] do
+      Mix.raise("""
+      [Bakeware] setting :#{option} in the release options outside of the :bakeware key is no longer supported.
+
+      Please adjust your release options in mix.exs to continue:
+
+        def release do
+          [
+            ...
+            steps: [&Bakeware.assemble/1, :assemble],
+            bakeware: [#{option}: #{inspect(value)}]
+            ...
+          ]
+        end
+      """)
+    end
+  end
```

Comment:
If you add an `:ok` at the end here, then you don't need to ignore the return value in the calls above.